packages/cli/README.md

```diff
@@ -233,1620 +232,2101 @@ get a specific app or a list of apps
 
 ```
 USAGE
-  $ smartthings apps [ID]
+  $ smartthings apps [ID] [-h] [-p <value>] [-t <value>] [--language <value>] [--indent <value>] [-j] [-y]
+    [-o <value>] [--compact] [--expanded] [--type <value>] [--classification <value>] [-v]
```

Comment:
The changes to readme output are from migrating away from `@oclif/dev-cli` to `oclif` instead.